### PR TITLE
feat(issue-322): Make the page tab and the action buttons same line

### PR DESCRIPTION
**Pull request by jisung-lee7**
## Issue Link

[issue-322](https://github.com/kafelix496/coding-for-fun/issues/322)

## Notes

-

### CODE REVIEW

**Review by jiyeol-lee — CHANGES_REQUESTED** [review]
Nope, I actually want to render these buttons next to switch button

File: src/elements/tabs/tabs.tsx [hunk 1]
```diff
@@ -90,13 +91,17 @@ export const Tabs = ({
...
+        <TabsList data-testid="tabs-list">
+          {values.map(({ label, value }) => (
+            <TabsTrigger data-testid="tabs-trigger" key={value} value={value}>
+              {label}
+            </TabsTrigger>
+          ))}
+        </TabsList>
+        {value === 'PULLS' &&
+          values.map(({ value, actions }) => <div key={value}>{actions}</div>)}
+      </div>
```
Comment (jiyeol-lee) [inline src/elements/tabs/tabs.tsx:104]: Nope, this is very bad way. You are making a core component like a specific domain component.

**Review by jiyeol-lee — CHANGES_REQUESTED** [review]

File: src/elements/tabs/tabs.tsx (hunk 1 above)
Comment (jisung-lee7) [reply src/elements/tabs/tabs.tsx]: Sorry, I forgot that I had to modify that.

File: src/app/(views)/github/t/layout.tsx [hunk 2]
```diff
@@ -37,37 +45,38 @@ export default function Layout({ children }: { children: ReactNode }) {
     {
       label: translate('GITHUB.TAB_CONNECTIONS_LABEL'),
       value: ETabValue.CONNECTIONS,
-      children
+      children: children
```
Comment (jiyeol-lee) [inline src/app/(views)/github/t/layout.tsx:48]: Just use one time. It's redundant to do like `children: children`.

File: src/app/(views)/github/t/layout.tsx [hunk 3]
```diff
@@ -37,37 +45,38 @@ export default function Layout({ children }: { children: ReactNode }) {
...
     },
     {
       label: translate('GITHUB.TAB_BULK_PULL_REVIEWS_LABEL'),
       value: ETabValue.PULLS,
-      children
+      children: children,
```
Comment (jiyeol-lee) [inline src/app/(views)/github/t/layout.tsx:53]: > Just use one time. It's redundant to do like `children: children`.

Same here

**Review by jiyeol-lee — APPROVED** [review]

**Pull request merged**